rest_framework_json_api/renderers.py

```diff
@@ -216,9 +223,15 @@ def extract_relationships(cls, fields, resource, resource_instance):
                         utils.get_resource_type_from_instance(nested_resource_instance)
                     )
 
+                    if hasattr(field.child_relation, 'lookup_field'):
```

Comment:
couldn't this code block be simplified if we add a `lookup_field` to `ResourceRelatedField` defaulting to `pk`?